_source/logzio_collections/_log-sources/aws-security-hub.md

```diff
@@ -0,0 +1,83 @@
+{:.info-box.important}
+<!-- info-box-end -->
+
+This deployment will automatically create the following resources:
+
+![Resources](https://dytvr9ot2sszz.cloudfront.net/logz-docs/aws/resources-security-hub.png)
+
+This integration ships events from AWS Security Hub to Logz.io.
+
+A new event triggers a designated EventBridge rule, which invokes a Lambda function. The function processes the event and sends it to Logz.io.
+
+<div class="tasklist">
+
+##### Login to your account
+
+To begin, you need to login to your AWS account.
+
+##### Create a new stack
+
+Select the button below to create a new stack dedicated to sending events from AWS Security Hub to Logz.io.
+
+| AWS Region | Launch a stack |
+| --- | --- |
+| `us-east-1` | [![Deploy to AWS](https://dytvr9ot2sszz.cloudfront.net/logz-docs/lights/LightS-button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=logzio-security-hub-collector&templateURL=https://logzio-aws-integrations-us-east-1.s3.amazonaws.com/aws-security-hub-collector/0.0.1/template.yaml) |
+
+![Create stack](https://dytvr9ot2sszz.cloudfront.net/logz-docs/aws/security-hub-step1.png)
+
+Keep the default setting in the **Create stack** screen and select **Next**.
+
+##### Specify the stack details
+
+![Specify stack details](https://dytvr9ot2sszz.cloudfront.net/logz-docs/aws/security-hub-step2.png)
+
+Specify the stack details as per the table below and select **Next**.
+
+
+| Parameter | Description |
+| --- | --- |
+| `logzioListener` | Your Logz.io [listener url](https://docs.logz.io/user-guide/accounts/account-region.html), followed by port `8070` or `8071`. For example, `https://listener.logz.io:8071` |
+| `logzioLogLevel` | Log level for the Lambda function. Defaults to `info`. Valid options are: `debug`, `info`, `warn`, `error`, `fatal`, `panic`. |
+| `logzioOperationsToken` | Your Logz.io [operations token](https://app.logz.io/#/dashboard/settings/general). |
+
+##### Configure the stack options
+
+![Configure stack options](https://dytvr9ot2sszz.cloudfront.net/logz-docs/aws/security-hub-step3.png)
+
+Specify the **Key** and **Value** parameters for the **Tags** and select **Next**.
+
+##### Review the deployment
+
+![Review deployment](https://dytvr9ot2sszz.cloudfront.net/logz-docs/aws/security-hub-step4a.png)
+![Confirm deployment](https://dytvr9ot2sszz.cloudfront.net/logz-docs/aws/security-hub-step4b.png)
+
+Confirm that you acknowledge that AWS CloudFormation might create IAM resources and select **Create stack**.
+
+##### Check Logz.io for your logs
+
+Give your logs some time to get from your system to ours, and then open [Kibana](https://app.logz.io/#/dashboard/kibana).
+
+If you still don't see your logs, see [log shipping troubleshooting]({{site.baseurl}}/user-guide/log-shipping/log-shipping-troubleshooting.html).
```

Comment:
A few points:
* Can we refer to it as "events" rather than "logs"?
* I think it would be more accurate if we will mention that it also takes a few minutes for the stack to be deployed, and for the resources to be created.
* The events won't be shipped as soon as the stack is created, so "Give your logs some time to get from your system to ours" may not be so accurate. Once the stack is created it waits for a new event. So in theory, the stack can be deployed, but if let's say only an hour later a new event was created in the security hub, only then it will appear in Logz.io. How can we rephrase?